NEWS.md

```diff
@@ -1,6 +1,7 @@
 # sfdep 0.2.4
 
-- Dexter Locke is now the maintainer of sfdep
+- updated spacetime_constructor to comply with updates to the cli package. No user-facing changes
```

Comment:
this needs to be under a new version heading since 0.2.4 is released on CRAN. This bug is not in version 0.2.4. 

Can you increment the version using `usethis::use_version("dev")`? Then move this note under the new version heading it will make?